webapp/loggedin/admin/actions.js

```diff
@@ -20,8 +20,8 @@ export const getVersions = () => dispatch => {
 export const createVersion = (e) => (dispatch, getState) => {
   const state = getState();
   const { newVersionForm } = state.admin
-  // Required fields for new version: version number and timestamp
-  const validForm = newVersionForm && newVersionForm.version && newVersionForm.timestamp
+  // Required fields for new version: version number and publishedAt
+  const validForm = newVersionForm && newVersionForm.versionNumber && newVersionForm.publishedAt
```

Comment:
can we use the FraVersion object to access versionNumber and publishedAt fields?